README.md

```diff
@@ -0,0 +1,85 @@
+| openvpn_proto                      | `udp`                                             | The protocol you want OpenVPN to use                                                                                                                            |
+| openvpn_dualstack                  | `true`                                            | Whether or not to use a dualstack (IPv4 + v6) socket                                                                                                            |
+| openvpn_config_file                | `openvpn_{{ openvpn\_proto }}_{{ openvpn_port }}` | The config file name you want to   use                                                                                                                          |
+| openvpn_rsa_bits                   | `2048`                                            | Number of bit used to protect generated certificates                                                                                                            |
+| openvpn_service_name               | `openvpn`                                         | Name of the service. Used by systemctl to start the service                                                                                                     |
+| openvpn_uninstall                  | `false`                                           | Set to true to uninstall the OpenVPN service                                                                                                                    |
+| openvpn_use_pregenerated_dh_params | `false`                                           | DH params are generted with the install by default                                                                                                              |
+| openvpn_use_modern_tls             | `true`                                            | Use modern Cipher for TLS encryption                                                                                                                            |
+| openvpn_verify_cn                  | `false`                                           | Check that the CN of the certificate match the FQDN                                                                                                             |
+| openvpn_redirect_gateway           | `true`                                            | OpenVPN gateway push                                                                                                                                            |
+| openvpn_set_dns                    | `true`                                            | Will push DNS to the client (Google and OpenDNS)                                                                                                                |
```

Comment:
Does this mean that you push Google DNS and OpenDNS?

Reply:
fixed